src/server/routes/proxy.js

```diff
@@ -60,6 +61,19 @@ export default ({ dataStore, proxyConfig, socketIo }) => {
     try {
       res.json(JSON.parse(mock.responseBody));
     } catch (err) {
+      const extension = path.extname(url).substring(1);
```

Comment:
It would be good to do this based on HTTP headers (Content-Type or Accept) but it would require more changes (e.g. saving the content type on the mock based on HTTP header and using that here and possibly adding a field in the UI to be able to specify that).

Reply:
Interesting idea. Maybe we can come back to it at some point.